dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/JdbcPropBuilder.java

```diff
@@ -77,7 +75,11 @@ public JdbcPropBuilder prop(@Nonnull String prop, @Nonnull String val) {
     }
 
     private void addProp(String prop, String val) {
-        props.add(prop + punctuations.charAt(1) + URLEncoder.encode(val, UTF_8));
+        try {
+            props.add(prop + punctuations.charAt(1) + URLEncoder.encode(val, "UTF-8"));
+        } catch (UnsupportedEncodingException e) {
+            LOG.warn("Could not encode {}={} into JDBC URL, the property will be dropped.", prop, val);
```

Comment:
This looks like a semantic change. We called a method, and it did nothing but log the fact that it didn't do what was asked. Why do we need this? What invalid parameter are we trying to pass and ignore?

Reply:
Right, probably just throwing and make the dumper fail would make more sense.